### PR TITLE
Update to airlift 241 and Jetty 12.0.7

### PR DESCRIPTION
**Pull request by wendigo**
Requires https://github.com/airlift/airlift/pull/1089

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1671,6 +1663,12 @@
                 <version>8.5.13</version>
             </dependency>
 
+            <dependency>
```
Comment (electrum) [inline pom.xml:1666]: We should add this to Airbase

**Review by electrum — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (wendigo) [reply pom.xml:1666]: Agree. I'll cleanup after airlift is released and this PR is merged

**Comment by wendigo** [comment]
@electrum seems good to go. Only commit dispatcher has failed. Tests are stable. I've ran several rounds of benchmarks on that PR and didn't met any stability issues in the current configuration

File: pom.xml (hunk 1 above)
Comment (wendigo) [reply pom.xml:1666]: https://github.com/airlift/airbase/pull/397

**Pull request merged**